spec/support/webmock.rb

```diff
@@ -25,7 +25,3 @@ def to_return_fm(hash_or_status = {}, options = {})
 
 # Don't raise but report uncaught net connections
 WebMock.disable_net_connect!
-WebMock.stub_request(:any, /.*/).to_return do |request|
-  puts "\e[35mUNSTUBBED REQUEST:\e[0m #{request.method.upcase} #{request.uri}"
-  { body: nil }
-end
```

Comment:
WebMock provides very useful error messages if you let it throw errors with `WebMock.disable_net_connect!`. I removed this code so I could see those errors and diagnose why my specs were failing. I can put this back in if it's needed...